qds_sdk/commands.py

```diff
@@ -136,6 +150,19 @@ def get_log(self):
         r = conn.get_raw(log_path)
         return r.text
 
+    def get_log_partial(self, err_pointer=0, tmp_pointer=0):
+        """
```

Comment:
dont want to mess around with get_log method, as it might be used by others who build their things on top of sdk